pysindy/feature_library/base.py

```diff
@@ -451,7 +454,7 @@ def transform(self, x_full):
 
         xp_full = []
         for x in x_full:
-            xp = []
+            xp = None
             for i in range(len(self.libraries)):
                 lib_i = self.libraries[i]
                 if self.inputs_per_library is None:
```

Comment:
`self.inputs_per_library` is always set in [`TensoredLibrary.fit()`](https://github.com/dynamicslab/pysindy/blob/c1da1f9e25b008e2f3016f2462ce100d739ea3f9/pysindy/feature_library/base.py#L411), and the function begins with ` check_is_fitted`.  So this conditional isn't necessary.